bootstrap/eks/hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Is this file needed, or can any scripts referencing it be updated to use the one in the root of the repo?